packages/kit/src/core/server/index.js

```diff
@@ -1,25 +1,25 @@
 import http from 'http';
 import https from 'https';
+//import config from '../../../test/apps/amp/svelte.config';
 /**
  *
  * @param {boolean} use_https
- * @param {any} user_config
+ * @param {import('types/config').ViteConfig} vite_config
  * @param {(req: http.IncomingMessage, res: http.ServerResponse) => void} handler
  * @returns {Promise<import('net').Server>}
  */
-export async function get_server(use_https, user_config, handler) {
+export async function get_server(use_https, vite_config, handler) {
 	/** @type {https.ServerOptions} */
-	const https_options = {};
+	const https_options =
+		vite_config && vite_config.server && vite_config.server.https === 'object'
```

Comment:
This line doesn't make sense to me, you're saying `vite_config.server.https === 'object'` which means it's a literal string, but then trying to destructure. Did you mean to include a `typeof`?

Reply:
You're right. Fixed in latest commit(s).  I'm still getting the same output for "pnpm run dev" and "pnpm run build":
```
 The value for kit.vite.server.https.key.toString specified in svelte.config.js has been ignored. This option is controlled by SvelteKit.
  The value for kit.vite.server.https.key.toLocaleString specified in svelte.config.js has been ignored. This option is controlled by SvelteKit.
  The value for kit.vite.server.https.cert.toString specified in svelte.config.js has been ignored. This option is controlled by SvelteKit.
  The value for kit.vite.server.https.cert.toLocaleString specified in svelte.config.js has been ignored. This option is controlled by SvelteKit.
```

Reply:
I'm concerned that this PR might be accidentally merged -- I haven't been able to perform my own testing to this PR.

Is there any way to denote this in the PR?  Should I have done things differently?

Reply:
I'm able to repro the issue, working on a fix now.

Reply:
A quick fix that I verified solves your issue is here: https://github.com/sveltejs/kit/pull/1698